### PR TITLE
Archive update v0.53.2

### PR DESCRIPTION
**Pull request by BytewaveMLP**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,529 +1,2 @@
...
-#### v0.35.1
-- Fixed performance issues with ignore system and hiding system
-- Fixed performance issues when a lot of users are joining the server at the same time
-
-#### v0.35.0
-- Added flying - <kbd>C</kbd> key or <kbd class="gamepad gamepad-dpad"><div class="dpad-up"></div></kbd> button or <code>/fly</code> command
-- Added booping while flying
-- Added flying above water
+#### v0.0.1
+- Initial release
```
Comment (Stubenhocker1399) [inline CHANGELOG.md]: We're starting from scratch?

Comment (BytewaveMLP) [reply CHANGELOG.md]: This is how the PT Team left theirs. I've started working on an updated changelog on another branch describing our changes.

It seems like most people are dumping Pony Town's changelogs and writing their own.

File: src/ts/components/shared/install-button/install-button.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ export class InstallButton {
 	constructor(private installService: InstallService) {
 	}
 	get canInstall() {
-		return this.installService.canInstall;
+		return false;
```
Comment (Stubenhocker1399) [inline src/ts/components/shared/install-button/install-button.ts]: Strange, are custom servers not allowed to have app like experiences?

Comment (BytewaveMLP) [reply src/ts/components/shared/install-button/install-button.ts]: The install functionality has apparently been removed from Pony Town proper.

File: .gitignore [hunk 3]
```diff
@@ -1,14 +1,12 @@
...
 node_modules/
 
-/src/scripts/
 /build/
 /build-copy/
 /logs/
 /settings/
 
-/tools/
-/store/
```
[the remark this answers is not in the document]
Comment (BytewaveMLP) [reply .gitignore]: At least `/store/` is `.gitignore`d on its own, in such a way that the directory is made on checkout.

File: config-template.json [hunk 4]
```diff
@@ -8,8 +8,9 @@
 	"host": "http://localhost:8090/",
 	"local": "localhost:8090",
 	"adminLocal": "localhost:8091",
+	"proxy": false,
 	"secret": "<some_random_string_here>",
-	"token": "<some_random_string_here>",
+	"token": "<some_other_random_string_here>",
```
[the remark this answers is not in the document]
Comment (BytewaveMLP) [reply config-template.json]: Theoretically, if this hasn't been changed, it's unlikely `secret` will have changed either. But yes, the check should be updated to do this in another commit.

File: .gitignore (hunk 3 above)
[the remark this answers is not in the document]
Comment (Stubenhocker1399) [reply .gitignore]: Ah, alright.

File: webpack.dev.js [hunk 5]
```diff
@@ -18,6 +18,7 @@ module.exports = merge(common, {
 	devtool: 'cheap-eval-source-map',
 	devServer: {
 		host: '0.0.0.0',
+		port: 8090,
```
Comment (Stubenhocker1399) [inline webpack.dev.js]: Looks like this might be another port we might want to get from the config file (#92).

**Pull request merged**